### PR TITLE
Updated README

### LINKED ISSUE #90: Update README
- Remove members that have left/ Add new members
- Update setup instructions
- Perhaps a more detailed description of our project

### PR DESCRIPTION
**Pull request by gauravnv**
Check it out and ask for updates if needed.

Closes #90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,31 +9,43 @@ The main functionalities:
...
+
+## 2) Getting the project
+
+Clone the repo:
 - **HTTPS** `git clone https://github.com/CodeTheChangeUBC/ReBOOT.git`
 - **SSH** `ssh git@github.com:CodeTheChangeUBC/ReBOOT.git`
 
+Install Requirements:
+- `cd project_directory`
+- `pip install requirements.txt`
```
Comment (leesw98) [inline README.md]: `pip install -r requirements.txt`

File: README.md [hunk 2]
```diff
@@ -9,31 +9,43 @@ The main functionalities:
...
+    - Check out a tutorial if you're unsure
+
+## 2) Getting the project
+
+Clone the repo:
 - **HTTPS** `git clone https://github.com/CodeTheChangeUBC/ReBOOT.git`
 - **SSH** `ssh git@github.com:CodeTheChangeUBC/ReBOOT.git`
 
+Install Requirements:
+- `cd project_directory`
```
Comment (leesw98) [inline README.md]: Remove and add `brew install rabbitmq` (one for windows too)

File: README.md [hunk 3]
```diff
@@ -46,20 +58,38 @@ When making a **contribution**, make sure to do the following:
 5. `git commit -m "ADD A DESCRIPTION OF YOUR CHANGES"` to commit your changes to the branch
 6. `git push origin [name_of_your_branch]` to push your changes to the repo
 7. Create a **Pull Request** into the master branch
-8. After approval from **at least one** supervisor (**@leesw98** or **@ying-choi**) merge branch into master.
+8. After approval from **at least one** supervisor (**@leesw98** or **@michellehuh** or **gauravnv**) merge branch into master.
```
Comment (leesw98) [inline README.md]: Maybe you wanted to @gauravnv

File: README.md [hunk 4]
```diff
@@ -46,20 +58,38 @@ When making a **contribution**, make sure to do the following:
...
 
 ## AWS RDS MySQL Connection
 
 If you wanted to manually connect to the database using the terminal, you can do the following.
 
-`mysql -h rebootdbinstance.cn0ttbkdpgt2.ca-central-1.rds.amazonaws.com -u ctc_reboot -p`
+~~`mysql -h rebootdbinstance.cn0ttbkdpgt2.ca-central-1.rds.amazonaws.com -u ctc_reboot -p`~~
```
Comment (leesw98) [inline README.md]: Delete it. We don't have this db anymore

File: README.md (hunk 2 above)
Comment (leesw98) [reply README.md]: Also postgres app for mac and idk what for window

Comment (gauravnv) [reply README.md]: Got it!

**Comment by gauravnv** [comment]
@leesw98 review ready

**Pull request merged**